linkatos/message.py

```diff
@@ -13,3 +15,16 @@ def contains_a_link(message):
         answer = answer.group(0).strip()
 
     return answer
```

Comment:
I'd rename this function to `extract_url` because it's what you are actually doing.